styles/index.css

```diff
@@ -17,10 +17,19 @@ h2 {
   margin: 2rem 0;
```

Comment:
suggestion - add comments for different sections inside the file, if the code length is more it is easy to find a particular section.

Reply:
This is automatically generated